package.json

```diff
@@ -26,7 +26,7 @@
   "homepage": "https://github.com/digitalbazaar/oauth2-token-endpoint-handler",
   "dependencies": {
     "@interop/oauth2-errors": "^2.0.0",
-    "assert-plus": "^1.0.0"
```

Comment:
I removed it assuming that we'll be using this module in both the client and server, and if we use assert-plus it won't work in client since it's a nodejs library.